UserDataDeleteOrchestrator/__tests__/handler.test.ts

```diff
@@ -191,6 +191,7 @@ describe("createUserDataDeleteOrchestratorHandler", () => {
     expect(OrchestratorFailure.decode(result).isRight()).toBe(true);
     expect(setUserDataProcessingStatusActivity).toHaveBeenCalledWith(
       expect.any(String),
+      expect.any(Object),
```

Comment:
Tip: to make the test more eloquent, we can give a name to this parameter:
```ts
const expectedRetryOptions = expect.any(Object);
```
 and then use it like
 ```ts
  expect(setUserDataProcessingStatusActivity).toHaveBeenCalledWith(
      expect.any(String),
      expectedRetryOptions,
      expect.objectContaining({
        nextStatus: UserDataProcessingStatusEnum.FAILED
      })
  )
```

Reply:
Nice tip!